scripts/ciTest.js

```diff
@@ -39,8 +39,8 @@ if (all){
     installGlobal = true
     ounitTest = true
     mochaTest = true
-    themeTest = true
-    bsbTest = true
+    // themeTest = true
+    // bsbTest = true
```

Comment:
This does not work because `npm link` does not work under Azure?

Reply:
Yes that's my guess. I think links in general are problematic.
So I'm skipping this for now as I'm more interested in getting Windows to work.
The other observation is that CircleCI is definitely faster on Linux.